config/database.yml.sample

```diff
@@ -9,9 +9,9 @@ default: &default
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  username: <%= ENV.fetch("POSTGRES_USER") { `echo $USER`} %>
-  password: <%= ENV.fetch("POSTGRES_PASSWORD") { "" } %>
-  host: <%= ENV.fetch("POSTGRES_HOST") { "" } %>
+  username: <%= ENV.fetch("POSTGRES_USER") { "postgres" } %>
+  password: <%= ENV.fetch("POSTGRES_PASSWORD") { "postgres" } %>
+  host: <%= ENV.fetch("POSTGRES_HOST") { "localhost" } %>
```

Comment:
@arielj : We don't have POSTGRES_USER and POSTGRES_PASSWORD in the `.env.sample` yet.

Reply:
@neutrino but those variables are being set by the docker-compose.yml file, though the docker-compose.yml file should also set the POSTGRES_HOST variable

Docker is setting variables so I don't think we need to change the fallback since we can set the variables